elevate-survey/controllers/samiksha.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * name : controllers/project.js
+ * author : Adithya Dinesh
+ * Date : 22-Aug-2024
+ * Description : Orchestration controller for Samiksha service
+ */
+
+const routeConfigs = require('../constants/routes')
+const requesters = require('../utils/requester')
+const common = require('../constants/common')
+
+/**
+ * Fetch resources from Samiksha service.
+ * @name fetchSamikshaTemplates
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} responses  
+ * @returns {JSON} - List of resources
+ */
+/**
+ * req.body = {
+ * 		organization_id : 1,
+ * 		resourceType : ['survey','projects']
+ * }
+ */
+
+
+const fetchSamikshaTemplates = async (req, res, responses) => {
+	const selectedConfig = routeConfigs.routes.find((obj) => obj.sourceRoute === req.sourceRoute)
+
+	let response = {result: {data: [],count: 0}}
+	let proceedToCallProjectService = true
+	let resp = {}
```

Comment:
@adithyadinesh0412 as same as I commented in projects it can be false no need to call service if data is empty